### ISSUE TITLE
Bug when setting borderWidth to 0 programatically on iOS

### ISSUE THREAD

**Issue by bnussey** [NativeScript/NativeScript#3164]
### Did you verify this is a real problem by searching [Stack Overflow]
Yes. 

### Tell us about the problem
In my project I programmatically set `borderWidth` to 5 upon item selection. When item is unselected I set `borderWidth` to 0 which should result in border disappearing. This occurs on Android, but not on iOS.  This was working prior to `2.4.0`

```
let imageElement = this._page.getViewById(id);
imageElement.borderColor = new Color("#ff0000");
if (value === true) {
    imageElement.borderWidth = 5;
}
else {
    imageElement.borderWidth = 0;
}
```
### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- CLI: `2.4.0`
- Cross-platform modules: `2.4.0`
- Runtime(s): "tns-ios": "2.4.0", "tns-android": "2.4.1"

**Comment by zh-m** [comment]
Hi @bnussey,

I resolved the issue with the following merge  -  #3167 https://github.com/NativeScript/NativeScript/pull/3167

**Comment by bnussey** [comment]
Thanks @zh-m
